cdk/script/tsc

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I think under the [scripts pattern](https://github.com/github/scripts-to-rule-them-all) this file would be `script/build`. I think the key is using agnostic language that applies to any repository to provide a common experience.